x-pack/plugins/elastic_assistant/server/routes/evaluate/utils.ts

```diff
@@ -23,7 +23,7 @@ export const getLlmType = (connectorId: string, connectors: ActionResult[]): str
     // See: https://github.com/langchain-ai/langchainjs/blob/fb699647a310c620140842776f4a7432c53e02fa/langchain/src/agents/openai/index.ts#L185
     return 'openai';
   }
-  // TODO: Add support for AWS Bedrock Connector once merged
+  // TODO: Add support for Amazon Bedrock Connector once merged
```

Comment:
Reminded @spong about this note 😄

Reply:
Ty! For posterity, the evaluator will still work with the Amazon Bedrock Connector, we just need to make a custom agent wrapper if we want to leverage the experimental function calling (or any other specific behavior on the agent side).